CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## [v2.2.0] - 2021-06-28
+
+### Changed
+
+- *: Implement GSP-97, GSP-109 and GSP-117 (#126)
```

Comment:
Maybe we need to extend them to:

- Implement GSP-97: xxxxx
- Implement GSP-109: xxxxx
- Implement GSP-117: xxxx